swagger.json

```diff
@@ -5642,6 +5622,62 @@
                 }
             }
         },
+        "VehicleResponse": {
+            "type": "object",
+            "description": "Contains information about the vehicle",
+            "properties": {
+                "id": {
+                    "type": "integer",
+                    "description": "ID of the vehicle.",
+                    "format": "int64",
+                    "example": 123
+                },
+                "name": {
+                    "type": "string",
+                    "description": "Name of the vehicle",
+                    "format": "string",
+                    "example": "Bob's Truck"
+                },
+                "make": {
+                    "type": "string",
+                    "description": "Make of the vehicle",
+                    "format": "string",
+                    "example": "Toyota"
+                },
+                "model": {
+                    "type": "string",
+                    "description": "Model of the vehicle",
+                    "format": "string",
+                    "example": "Corolla"
+                },              
+                "vin": {
+                    "type": "string",
+                    "description": "VIN of the vehicle",
+                    "format": "string",
+                    "example": "1HGBH41JXMN109186"
+                },
+                "year": {
+                    "type": "integer",
+                    "description": "Year the vehicle was made.",
+                    "format": "int64",
+                    "example": "2017"
+                },     
+                "harshAccelSetting": {
+                    "type": "integer",
+                    "description": "A number from 1-10 indicating the harsh acceleration setting of the vehicle",
+                    "format": "int64",
+                    "example": "2017"
+                },                            
+                "externalIds" : {
+                    "type": "object",
+                    "description": "Dictionary of external IDs (string key-value pairs)",
+                    "additionalProperties": {
+                      "type": "string"
+                    },
+                    "example": {"internalVehicleId": "ABFS18600"}
```

Comment:
Nit: it's a little confusing to have a key `internalVehicleId` in the dictionary `externalIds`.